handlers/map-tactile-svg/map-svg.py

```diff
@@ -0,0 +1,292 @@
+        return response
+
+    dimensions = 700, 700
+    remove_streets = ["footway", "crossing", "steps"]
+    svg = draw.Drawing(dimensions[0], dimensions[1])
+
+    data = preprocessor["ca.mcgill.a11y.image.preprocessor.openstreetmap"]
+    if "streets" in data:
+        streets = data["streets"]
+        lat = data["bounds"]["latitude"]
+        lon = data["bounds"]["longitude"]
+        lon_min = lon["min"]
+        lat_min = lat["min"]
+        lon_max = lon["max"]
+        lat_max = lat["max"]
+        # Scale the lon/lat units to svg pixels equivalent.
+        scaled_longitude = dimensions[0] / (lon_max - lon_min)
+        scaled_latitude = dimensions[1] / (lat_max - lat_min)
+
+        colors = [
+                  "red",
+                  "blue",
+                  "springgreen",
+                  "deeppink",
+                  "orange",
+                  "purple",
+                  "cyan",
+                  "coral",
+                  "teal",
+                  "indigo",
+                  "lime",
+                  "chocolate",
+                  "magenta",
+                  "crimson",
+                  "deepskyblue",
+                  "greenyellow",
+                  "gold",
+                  "green",
+                  "aqua",
+                  "navy",
+                  "royalblue",
+                  "forestgreen",
+                  "dodgerblue"
+                  ]
+        # Draw the streets with svg.
+        checkPOIs = {}
+        g = draw.Group(data_image_layer="firstLayer", aria_label="Streets")
+        for i, street in enumerate(streets):
+            color = i
+            if i >= len(colors):
+                color = i % len(colors)
+            # Filter only necessary street types
+            if street["street_type"] not in remove_streets:
+                name = street["street_name"] if "street_name"\
+                    in street else street["street_type"]
+                description = getDescriptions(street)
+                stroke_width = return_stroke_width(
+                    street["street_type"])
+                args = dict(stroke=colors[color], stroke_width=stroke_width,
+                            fill='none', aria_label=name)
+                # Add this arg only if the  not empty
+                if description is not None:
+                    args["aria_description"] = description
+                p = draw.Path(**args)
+                node_coordinates = []
+                for node in street["nodes"]:
+                    node_coordinates.append([node["lon"], node["lat"]])
+                    if "POIs_ID" in node:
+                        for x in node["POIs_ID"]:
+                            if x in checkPOIs:
+                                checkPOIs[x].append([name, description])
+                            else:
+                                checkPOIs[x] = [[name, description]]
+                for index in range(len(node_coordinates) - 1):
```

Comment:
Heads up that this is using the old iterator format! Unsure if you wanted to get all of these or not.

Reply:
Leaving this in with a less cryptic variable name called 'POI_ID' since this variable is used as a key for the dict 'checkPOIs'.